tools/db_cleanup.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+# Simple app that runs DB cleanup commands, and repopulates from disk
+ROOT_PASSWORD=${MYSQL_PASSWORD:StackStorm}
+
+
+echo "Cleaning MongoDB Database..."
+mongo st2 --eval "db.dropDatabase();"
+
+echo "Cleaning Mistral Database..."
+mysql -uroot -p$ROOT_PASSWORD -e "DROP DATABASE IF EXISTS mistral"
+mysql -uroot -p$ROOT_PASSWORD -e "CREATE DATABASE mistral"
```

Comment:
when mysql is set without password this will ask password 4 times, that's why I prefer to fire a single SQL statement with ; in between: `"DROP DATABASE IF EXISTS mistral;CREATE DATABASE mistra;.....l"`